### DISCUSSION TITLE
How does Material for MkDocs fetch the latest git tag (to display it in the navbar)?

### DISCUSSION THREAD

**Discussion by pawamoy** [squidfunk/mkdocs-material#3439] [Q&A] [answered]
We're having an issue with a site published at readthedocs.org: https://copier.readthedocs.io/en/latest/.
The git tag in the navbar shows 5.1.0, while the latest tag is 6.0.0a9.
I was wondering how the tag is obtained (upon building? with javascript?), and why it the latest tag is not used.
The [documentation](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#configuration) doesn't mention anything about this.
Any hints/ideas :blush: ?

**Comment by squidfunk** [comment, marked as answer]
https://github.com/squidfunk/mkdocs-material/blob/cbbef980dcb1e4a576b51f50ad7bf2560138b2f6/src/assets/javascripts/components/source/facts/github/index.ts#L65-L72

Note that we can only fetch the latest release, not tag. GitHub doesn't expose an API for obtaining the latest tag.

**Reply by pawamoy** [reply]
That makes total sense, thanks!

**Reply by yajo** [reply]
Is there a way to generate the version locally instead of pulling from github? 

Because I have several doc versions...

**Reply by squidfunk** [reply]
No, this is currently not supported.
